src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,32 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private Splitter splitter;
+    private Calculator calculator;
+    private EntryToFile entryToFile;
 
+    public WorkWithFile() {
+        this.splitter = new Splitter();
+        this.calculator = new Calculator();
+        this.entryToFile = new EntryToFile();
+    }
+
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The access modifier for `getStatistic` should be private if it's only used within this class, following the encapsulation principle.